packages/web/pages/dashboard.tsx

```diff
@@ -37,29 +35,6 @@ export interface ContainerQueries {
 }
 
 const ResponsiveGridLayout = WidthProvider(Responsive);
-export const gridData = [
-  { i: 'latest', x: 0, y: 0, w: 6, h: 6 },
-  { i: 'xp', x: 6, y: 0, w: 3, h: 2 },
-  { i: 'seed', x: 9, y: 0, w: 3, h: 2 },
-  { i: 'cal', x: 6, y: 2, w: 3, h: 4 },
-  { i: 'leaders', x: 9, y: 2, w: 3, h: 4 },
-];
-
-export const gridDataMd = [
-  { i: 'latest', x: 0, y: 0, w: 6, h: 4 },
-  { i: 'xp', x: 6, y: 0, w: 6, h: 2 },
-  { i: 'seed', x: 6, y: 2, w: 6, h: 2 },
-  { i: 'cal', x: 0, y: 4, w: 6, h: 4 },
-  { i: 'leaders', x: 6, y: 4, w: 6, h: 4 },
-];
-
-export const gridDataSm = [
-  { i: 'latest', x: 0, y: 3, w: 4, h: 3 },
-  { i: 'xp', x: 0, y: 0, w: 2, h: 2 },
-  { i: 'seed', x: 2, y: 0, w: 2, h: 2 },
-  { i: 'cal', x: 0, y: 5, w: 2, h: 4 },
-  { i: 'leaders', x: 2, y: 5, w: 2, h: 4 },
-];
 
 export const initLayouts = {
```

Comment:
Out of curiosity, why not move `initLayouts` out to `utils/dashboard.tsx` with the other layout constants and just have the one import for `initLayouts`?

Reply:
good remark, we can put at least all of the static values to the `utils/dashboard.tsx`, i missed this one. :)